odoo-shippable/scripts/library.sh

```diff
@@ -53,6 +53,16 @@ createuser_custom(){
     echo "${USER} ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/100-vauxoo-sudoers
     su - ${USER} -c "git config --global user.name ${USER}"
     su - ${USER} -c "git config --global user.email ${USER}@email.com"
+    if [ ${USER} == "odoo" ] || [ ${USER} == "root" ]; then
```

Comment:
The user `root` is not created because is created from OS.
What about move this bash change to `build-image.sh` script?